lib/transport/selenium-webdriver/index.js

```diff
@@ -193,17 +193,14 @@ class Transport extends BaseTransport {
 
   async createDriverService({options, moduleKey, reuseBrowser = false}) {
     try {
+      moduleKey = this.settings.webdriver.log_file_name || moduleKey || '';
 
       if (!this.shouldReuseDriverService(reuseBrowser)) {
         Transport.driverService = new this.ServiceBuilder(this.settings);
         await Transport.driverService.setOutputFile(reuseBrowser ? 'test' : moduleKey).init(options);
       }
 
       this.driverService = Transport.driverService;
-
-      moduleKey = this.settings.webdriver.log_file_name || moduleKey || '';
-
-      Transport.driverService = this.driverService;
```

Comment:
This change corrects a few things missed while making this commit: https://github.com/nightwatchjs/nightwatch/commit/e70e17d3dc395f413fc91cd4e5fd04255b9d861e